crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -2,26 +2,29 @@ use anyhow::{
     Context,
     Result,
 };
-use astria_proto::native::sequencer::v1alpha1::Address;
 use async_trait::async_trait;
 use borsh::{
-    BorshDeserialize as _,
-    BorshSerialize as _,
+    BorshDeserialize,
+    BorshSerialize,
 };
 use hex::ToHex as _;
 use penumbra_storage::{
     StateRead,
     StateWrite,
 };
+use proto::native::sequencer::v1alpha1::Address;
 use tracing::{
     debug,
     instrument,
 };
 
-use crate::accounts::types::{
-    Balance,
-    Nonce,
-};
+/// Newtype wrapper to read and write a u32 from rocksdb.
+#[derive(BorshSerialize, BorshDeserialize, Debug)]
+struct Nonce(u32);
```

Comment:
`Nonce` and `Balance` are now private to this module and just wrappers to ensure that they are correctly read from and written to rocksdb.